docs/quick-release-notes/main.go

```diff
@@ -0,0 +1,314 @@
+}
+
+func getTagsFromAPI(tagsRepoPath string) []tagInfo {
+	// https://docs.github.com/en/rest/reference/repos#list-repository-tags
+	tagsRepoOwner, tagsRepoName := splitAndEncodeURLPath(tagsRepoPath)
+	pathToTags := fmt.Sprintf("https://api.github.com/repos/%v/%v/tags", tagsRepoOwner, tagsRepoName)
+	resp, err := http.Get(pathToTags)
+	if err != nil {
+		log.Fatal(err)
+	}
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		log.Fatal(err)
+	}
+	responseFromAPI := []byte(body)
+	var tags []tagInfo
+	err = json.Unmarshal(responseFromAPI, &tags)
+	if err != nil {
+		log.Fatal(err)
+	}
+	return tags
+}
+
+func getChangeLogInfo(scanner *bufio.Scanner) changeLogInfo {
+	re, err := regexp.Compile(`## (?P<versionNumberAndDate>(?P<versionNumber>\d+\.\d+\.\d+).*)`)
+	if err != nil {
+		log.Fatal(err)
+	}
+	lineCount := 0
+	changeLogLength := 0
+	var versionLineNumbers []int
+	var changeLogText []string
+	var versionTitles []string
+	var versionNumbers []string
+	for scanner.Scan() {
+		changeLogLength++
+		changeLogText = append(changeLogText, scanner.Text())
+		if re.MatchString(scanner.Text()) {
+			hashVersionAndDate := re.FindStringSubmatch(scanner.Text())
+			idxVersionNumberAndDate := re.SubexpIndex("versionNumberAndDate")
+			idxVersionNumber := re.SubexpIndex("versionNumber")
+			versionNumberAndDate := hashVersionAndDate[idxVersionNumberAndDate]
+			versionNumber := hashVersionAndDate[idxVersionNumber]
+			versionTitles = append(versionTitles, versionNumberAndDate)
+			versionNumbers = append(versionNumbers, versionNumber)
+			versionLineNumbers = append(versionLineNumbers, lineCount)
+		}
+		lineCount++
+	}
+	if scanner.Err() != nil {
+		fmt.Println(scanner.Err())
+	}
+	return changeLogInfo{versionLineNumbers, changeLogText, changeLogLength, versionTitles, versionNumbers}
+}
+
+func createReleaseNotes(changeLogInfo changeLogInfo) []string {
+	re, err := regexp.Compile(`^\[.+\]: .+`)
+	if err != nil {
+		log.Fatal(err)
+	}
+	lastVersionIdx := len(changeLogInfo.versionLineNumbers) - 1
+	contributorsAndErrors := ""
+	for i := changeLogInfo.versionLineNumbers[lastVersionIdx]; i < changeLogInfo.changeLogLength; i++ {
+		if re.MatchString(changeLogInfo.changeLogText[i]) {
+			contributorsAndErrors += changeLogInfo.changeLogText[i] + "\n"
+		}
+	}
+	var releaseNotes []string
+	// exclude Last version (## 0.2.0) with - 1
+	for i, versionLineNumber := range changeLogInfo.versionLineNumbers[:] {
+		releaseBodyLines := ""
+		if !(i == (lastVersionIdx)) {
+			for j := changeLogInfo.versionLineNumbers[i] + 1; j < changeLogInfo.versionLineNumbers[i+1]; j++ {
+				releaseBodyLines += changeLogInfo.changeLogText[j] + "\n"
+			}
+		} else {
+			// Handle last version (## 0.2.0)
+			if versionLineNumber == changeLogInfo.versionLineNumbers[lastVersionIdx] {
+				for j := 1; j < changeLogInfo.changeLogLength-changeLogInfo.versionLineNumbers[lastVersionIdx]; j++ {
+					currentLineOfText := changeLogInfo.changeLogText[versionLineNumber+j]
```

Comment:
Nit: I think counting from `changeLogInfo.versionLineNumbers[i] + 1` (like line 249) or `versionLineNumber + 1` makes more sense:
```suggestion
				for j := versionLineNumber + 1; j < changeLogInfo.changeLogLength; j++ {
					currentLineOfText := changeLogInfo.changeLogText[j]
```